docs/config/events.md

```diff
@@ -49,11 +49,13 @@ Specifically, you need to hook into an event where the service you are running t
 Usage
 -----
 
-It's fairly straightforward to add events to your [Landofile](./lando.md) using the `events` top level config. Note that due to the nature of events eg automating steps that the *user* usually runs all event commands are run as "you" and do not have `sudo` or `root` access.
+It's fairly straightforward to add events to your [Landofile](./lando.md) using the `events` top level config.
 
-### Appserver commands
+Note that due to the nature of events eg automating steps that the *user* usually runs all event commands are run as "you" and do not have `sudo` or `root` access.
```

Comment:
Note that due to the nature of events, eg. automating steps that the *user* usually runs, all commands are run as "you" and do not have `sudo` or `root` access.

Reply:
makes sense i can fix that